change_quotes.py

```diff
@@ -104,6 +105,23 @@ def run_each(self, edit, sel_region):
         if not region:
             return
 
+        if self.custom:
+            fn_name = self.custom[0]
+            try:
+                fn = getattr(self, fn_name)
+            except AttributeError:
+                return
+
+            try:
+                fn_kwargs = self.custom[1]
+                if not isinstance(fn_kwargs, dict):
```

Comment:
I would expect this to fail (and show an error message in the console) rather than simply not work.